sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderer.kt

```diff
@@ -117,8 +117,7 @@ internal abstract class MapboxRenderer : MapClient {
     renderThread.queueEvent(runnable)
   }
 
-  @AnyThread
-  @Synchronized
+  @WorkerThread
```

Comment:
Changed annotation to indicate that main thread shouldn't call this blocking version of snapshot

Reply:
good catch